rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "nightly" # Allows us to use MappedLockGuard
```

Comment:
any chance of not using nightly channel? this would prevent brew from building the release.

Reply:
Thanks for pointing this out - fixed [here](https://github.com/thomasschafer/scooter/pull/50)